pyston/utils/compatibility.py

```diff
@@ -194,3 +194,10 @@ def render_template(template_name, context):
     if StrictVersion(django.get_version()) < StrictVersion('1.9'):
         context = Context(context)
     return get_template(template_name).render(context)
+
+
+def get_last_parent_pk_field_name(obj):
+    for field in obj._meta.fields:
+        if field.primary_key and (not field.is_relation or not field.auto_created):
+            return field.name
+    raise RuntimeError('Last parent field name was not fount (cannot happen)')
```

Comment:
docela sranda v djangu existuje struktura obj._meat.pk.name nicméně když má model parenta tak to vypíše např. user_ptr pro customera takže hledání reálného primárního klíče je trošku složitější, ale poradíme si. Ten runntime error je tam pro jistotu nikdy se nemůže stát jelikož každý model musí mít nějaké takové pole